### PR TITLE
Adds new integration [popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall]

### PR DESCRIPTION
**Pull request by popeen**
**Link to successful HACS action:**
https://github.com/popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall/actions/runs/3837723578/jobs/6533349919
**Link to successful hassfest action (if integration):**  
https://github.com/popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall/actions/runs/3837723578/jobs/6533350061

### CODE REVIEW

**Comment by popeen** [comment]
I am clearly missing something here, the validation in my repo is green but here I get "required key not provided @ data['issue_tracker']".

The issue tracker key is set as far as I can tell

Any suggestions for what I am doing wrong?

**Comment by ludeeus** [comment]
The action here (and HACS itself) uses the latest release.
https://github.com/popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall/blob/0.1.1/custom_components/motalavattenavfall/manifest.json

**Comment by popeen** [comment]
ah, of course, thank you!

File: integration [hunk 1]
```diff
@@ -618,6 +618,7 @@
   "plamish/xcomfort",
   "plmilord/Hass.io-custom-component-spaclient",
   "Poeschl/Remote-PicoTTS",
+  "popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall",
```
Comment (ludeeus) [inline integration:621]: This looks like a Swedish service, if it is, you should add `country` key in hacs.json

Comment (popeen) [reply integration:621]: Yea, its Swedish.
Just added the country key

**Pull request merged**